packages/marko-web-native-x/components/story/advertiser-related-stories.marko

```diff
@@ -0,0 +1,51 @@
+import { warn } from "@parameter1/base-cms-utils";
+import createClient from "../../apollo/create-client";
```

Comment:
This import is no longer needed if you're moving the Nx client creation to the config.